src/sdk/controllers/userApps.ts

```diff
@@ -152,6 +170,48 @@ export class UserApps {
         }
     }
 
+    /** Verifies the login requests, and checks if the apps have already been authorized with those keys
+     *
+     * @param {LoginRequest[]} requests - Array of login requests to check
+     * @returns {Promise<CheckedRequest[]>} - Array of requests that have been verified and had authorization checked
+     */
+    async checkRequests(requests: LoginRequest[]): Promise<CheckedRequest[]> {
```

Comment:
this function is currently unused, and was built for use https://github.com/Tonomy-Foundation/Tonomy-ID/issues/705